src/metpy/plots/patheffects.py

```diff
@@ -0,0 +1,208 @@
+    symbol = WarmFront.symbol
+    symbol2 = ColdFront.symbol.transformed(mtransforms.Affine2D().scale(1, -1))
+
+    def __init__(self, colors=None, **kwargs):
+        if colors is None:
+            colors = ['red', 'blue']
+        self._colors = list(map(mcolors.to_rgba, colors))
+        super().__init__(color=self._colors[0], **kwargs)
+
+    def _get_path_segment_ends(self, segment_offsets):
+        # Calculate increment of path length occupied by each marker drawn
+        inc = self.step_size
+
+        # Find out how many markers that will accommodate, as well as remainder space
+        num, leftover = divmod(segment_offsets[-1], inc)
+
+        # Find the offset for each path segment end. We center along
+        # the entire path by adding half of the remainder.
+        path_offsets = np.arange(1, num + 1) * inc + leftover / 2.
+
+        # Find the location of these offsets within the total offset within each
+        # path segment; subtracting 1 gives us the left point of the path rather
+        # than the last.
+        inds = np.searchsorted(segment_offsets, path_offsets)
+
+        # Return the indices to the proper segment and the offset within that segment
+        return inds
+
+    def draw_path(self, renderer, gc, path, affine, rgbFace=None):
+        gcs = [self._override_gc(renderer, gc, foreground=color) for color in self._colors]
+        self._gc_cycle = itertools.cycle(gcs)
+        self._symbol_cycle = itertools.cycle([self.symbol, self.symbol2])
+        self._color_cycle = itertools.cycle(self._colors)
+
+        # Get the information we need for drawing along the path
+        starts, offsets, angles = self._process_path(path, affine)
+
+        # Figure out what segments the markers should be drawn upon and how
+        # far within that segment the markers will appear.
+        segment_indices, marker_offsets = self._get_marker_locations(offsets)
+        end_path_inds = self._get_path_segment_ends(offsets)
+        start_path_inds = np.concatenate([[0], end_path_inds[:-1]])
+
+        # Set up proper scaling to take the markers in "points" and turn into pixels
+        base_trans = self._get_base_scale_transform(renderer)
+
+        # Loop over all the markers to draw
+        for ind, start_path, end_path, marker_offset in zip(segment_indices, start_path_inds,
+                                                            end_path_inds, marker_offsets):
+            azi = angles[ind]
+            sym_trans = base_trans.frozen().translate(
+                marker_offset - self.symbol_width * self.size / 2., 0).rotate(azi).translate(
+                *starts[ind])
+            gc = next(self._gc_cycle)
+            color = next(self._color_cycle)
+            symbol = next(self._symbol_cycle)
+
+            renderer.draw_path(gc, symbol, sym_trans, color)
+            renderer.draw_path(gc, mpath.Path(starts[start_path:end_path]),
+                               mtransforms.Affine2D(), color)
```

Comment:
```suggestion
                               mtransforms.Affine2D(), None)
```
Since this part is for the path piece of the stationary front, the `rgbFace` argument should just be `None`. This should get rid of the strange fill that I mentioned in the discussion.